workspaces/resource-optimization/plugins/resource-optimization/src/components/ExampleComponent/ExampleComponent.tsx

```diff
@@ -39,93 +38,107 @@ const SELECT_ITEMS = [
   },
 ];
 
-const columns: TableColumn<Recommendations>[] = [
-  {
-    title: 'Containers',
-    highlight: true,
-    render: row => <Link to="#message-source">{row.container}</Link>,
-  },
-  {
-    title: 'Projects',
-    render: row => <Typography variant="body2">{row.project}</Typography>,
-  },
-  {
-    title: 'Workloads',
-    render: row => <Typography variant="body2">{row.workload}</Typography>,
-  },
-  {
-    title: 'Workload types',
-    render: row => <Typography variant="body2">{row.workloadType}</Typography>,
-  },
-  {
-    title: 'Clusters',
-    render: row => <Typography variant="body2">{row.clusterAlias}</Typography>,
-  },
-  {
-    title: 'Last reported',
-    render: row => (
-      <Typography variant="body2">{row.lastReported?.toString()}</Typography>
-    ),
-  },
-];
 
 const ClusterFilter = () => (
-  <Select
-    placeholder="All results"
-    label="CLUSTER"
-    items={SELECT_ITEMS}
-    multiple
-    onChange={() => {}}
-  />
+  <>
+    <SearchBar
+        placeholder="Filter by Cluster"
+        debounceTime={700}
+    />
+    
+    <Chip
+      label='demo'
+      size='medium'
+      variant='default'
+      onDelete={() => ({})} />
+  </>
+  
 );
 
 const ProjectFilter = () => (
-  <Select
-    placeholder="All results"
-    label="PROJECT"
-    items={SELECT_ITEMS}
-    multiple
-    onChange={() => {}}
-  />
+  <>
+    <SearchBar
+        placeholder="Filter by Project"
+        debounceTime={700}
+    />
+  </>
 );
 
 const WorkloadFilter = () => (
-  <Select
-    placeholder="All results"
-    label="WORKLOAD"
-    items={SELECT_ITEMS}
-    multiple
-    onChange={() => {}}
-  />
+  <>
```

Comment:
Why these React.Fragments keep appearing where we don't need them? 😅

Reply:
sorry didn't focus much on cleanup for now, I have removed these and will keep cleaning up the code in upcoming PRs.